src/UI/Internal/Human.elm

```diff
@@ -0,0 +1,32 @@
+module UI.Internal.Human exposing (parseDate)
+
+
+parseDate : String -> Maybe ( Int, Int, Int )
```

Comment:
I think this is not safe enough. I would use this [package](https://package.elm-lang.org/packages/rtfeldman/elm-iso8601-date-strings/latest/) to do the conversion